FaceDetection/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 2.8.4)
+
+project(seeta_facedet_lib)
+
+# Build options
+option(BUILD_EXAMPLES  "Set to ON to build examples"  ON)
+option(USE_OPENMP      "Set to ON to build use openmp"  ON)
+
+# Use C++11
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
This is because he forgot to bind that c++11 as standard to an actual build target. CMake won't automagically know what and which target to compile with c++11 enabled. So something like this should do:

```
set_target_properties(seeta_facedet_lib PROPERTIES CXX_STANDARD 11)
set_target_properties(seeta_facedet_lib PROPERTIES CXX_STANDARD_REQUIRED ON) 
```